### PR TITLE
🛑     Add extension Stylelint

### LINKED ISSUE #2071: Install Stylelint

### PR DESCRIPTION
**Pull request by laliq**
Resolves #2071

### CODE REVIEW

**Review by mimarz — CHANGES_REQUESTED** [review]
Suggest also adding this extension to `.vscode/extensions.json` so that people not using codespaces also have it :)

File: .stylelint.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": [
+    "stylelint-config-recommended",
```
Comment (vnys) [inline .stylelint.json:3]:
```suggestion
```

The standard config [already extends recommended](https://github.com/stylelint/stylelint-config-standard/blob/main/index.js#L4) so we can remove this line.

**Review by vnys — CHANGES_REQUESTED** [review]

**Review by mimarz — CHANGES_REQUESTED** [review]
Stylelint out of the box only works for css or scss files I belive. Since most of our styling uses styled-components, we need to use a processor for the stylelint to work on _that_ styling.

https://styled-components.com/docs/tooling#stylelint

**Comment by mimarz** [comment]
Closing this as its quite outdated by now 👴

**Pull request closed without merging**